### PR TITLE
added doc for NERC RHOAI

### PR DESCRIPTION
**Pull request by Milstein**

### CODE REVIEW

File: docs/openshift-ai/data-science-project/explore-the-jupyterlab-environment.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+Make sure you are selecting "**mss-keycloak**" once shown:
+
+![RHOAI JupyterLab Login with KeyCloak](images/rhoai-jupyterlab-login.png)
+
+Authorize the requested permissions if needed:
+
+![Authorize Access to the RHOAI](images/authorize-access-to-the-rhoai.png)
+
+This will initiate your [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/)
+environment based on the Jupyter Image you have selected. JupyterLab offers an
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/explore-the-jupyterlab-environment.md:20]: offers an -> offers a

File: docs/openshift-ai/data-science-project/explore-the-jupyterlab-environment.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+shared interactive integrated development environment.
+
+Once you successfully authenticate you should see the NERC RHOAI JupyterLab Web
+Interface as shown below:
+
+![RHOAI JupyterLab Web Interface](images/jupyterlab_web_interface.png)
+
+It's pretty empty right now, though. So the first thing we will do is add content
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/explore-the-jupyterlab-environment.md:28]: So the -> The

File: docs/openshift-ai/data-science-project/explore-the-jupyterlab-environment.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+files by double-clicking them in the file explorer on the left side. Double-click
+the notebook file to launch it. This action will open another tab in the content
+section of the environment, on the right.
+
+Here, you can find three primary starter notebooks for setting up the intelligent
+application: `01_sandbox.ipynb`, `02_model_training_basics.ipynb`, and `03_remote_inference.ipynb`
+within the root folder path of `nerc_rhoai_mlops`.
+
+You can click and run `01_sandbox.ipynb` to verify the setup JupyterLab environment
+can run python codes properly.
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/explore-the-jupyterlab-environment.md:125]: codes -> code

File: docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+![Configure and Add A New Data Connection](images/configure-a-new-data-connection.png)
+
+For our example project, let's name it "ocp-nerc-container-connect", we'll select
+the "us-east-1" as **AWS_DEFAULT_REGION**, choose "ocp-container" as **AWS_S3_BUCKET**.
+The API Access EC2 credentials can be downloaded and accessed from the NERC OpenStack
+Project as [described here](../../openstack/persistent-storage/object-storage.md#configuring-the-aws-cli).
+This credential file contains information regarding **AWS_ACCESS_KEY_ID**,
+**AWS_SECRET_ACCESS_KEY**, and **AWS_S3_ENDPOINT**.
+
+**NOTE**: If you are using AWS S3 bucket then **AWS_S3_ENDPOINT** need to be set
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md:47]: need to -> needs to

File: docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+After completing the required fields, click **Add data connection**. You should
+now see the data connection displayed in the main project window as shown below:
+
+![New Data Connection Info](images/data-connection-info.png)
+
+## Create a model server
+
+After creating the data connection, you can configure your model server. Select
+**Configure server** as shown below:
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md:77]: the image says "Add Server" not "Configure sever".

File: docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+the complexity of your model.
+
+- **Model route**: Check this box if you want the serving endpoint (the model serving
+API) to be accessible outside of the OpenShift cluster through an external route.
+
+- **Token authorization**: Check this box if you want to secure or restrict access
+to the model by forcing requests to provide an authorization token.
+
+After adding and selecting options within the **Configure model server** pop-up
+window, click **Configure** to configure the model server.
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md:106]: I don't see Configure on the image above, I see Add?

File: docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+![Model Deployed Successfully](images/model-deployed-successful.png)
+
+The model is now accessible through the API endpoint of the model server. The
+information about the endpoint is different, depending on how you configured the
+model server.
+
+If you did not expose the model externally through a route, click on the Internal
+Service link in the Inference endpoint section. A popup will display the address
+for the gRPC and the REST URLs fpr the inference endpoints as shown below:
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/model-serving-in-the-rhoai.md:175]: fpr -> for

File: docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+4. Post-process the inference and send back the result
+
+5. Pass the result to the frontend for display
+
+### Deploy the application
+
+The deployment of the application is really easy, as we already created for you
+the necessary YAML files. They are included in the Git project we used for this
+example project. You can find them in the **deployment** folder inside you JupyterLab
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md:64]: inside you -> inside your

File: docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+environment, or directly [here](https://github.com/nerc-project/nerc_rhoai_mlops/tree/main/deployment).
+
+To deploy the Pre-Post Processing Service service and the Application:
+
+- From your [NERC's OpenShift Web Console](https://console.apps.shift.nerc.mghpcc.org/),
+navigate to your project corresponding to the *NERC RHOAI Data Science Project*,
+and select the "Import YAML" button, the "plus" sign on the top right:
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md:71]: add something about it being shown in the image below.

File: docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+- Once Resource is successfully created, you will see the following screen:
+
+    ![Resources successfully created Importing More YAML](images/yaml-import-new-content.png)
+
+- Click on "Import more YAML" and Copy/Paste the content of the file `intelligent_application_deployment.yaml`
+inside the opened YAML editor. Nothing to change here, you can then click the
+**Create** button as shown below:
+
+    ![YAML Editor Pre-Post Processing Service Content](images/intelligent_application_deployment-yaml-content.png)
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md:90]: this image is very small, it is hard to read.

File: docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+and select the "Import YAML" button, the "plus" sign on the top right:
+
+    ![YAML Add Icon](images/yaml-upload-plus-icon.png)
+
+- Copy/Paste the content of the file `pre_post_processor_deployment.yaml` inside
+the opened YAML editor. If you have named your model **coolstore** as instructed,
+you're good to go. If not, modify the value on **line # 35** with the name you set.
+You can then click the **Create** button as shown below:
+
+    ![YAML Editor Add Pre-Post Processing Service Content](images/pre_post_processor_deployment-yaml-content.png)
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md:80]: this image is very small, hard to read.

File: docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+
+- From your [NERC's OpenShift Web Console](https://console.apps.shift.nerc.mghpcc.org/),
+navigate to your project corresponding to the *NERC RHOAI Data Science Project*,
+and select the "Import YAML" button, the "plus" sign on the top right:
+
+    ![YAML Add Icon](images/yaml-upload-plus-icon.png)
+
+- Copy/Paste the content of the file `pre_post_processor_deployment.yaml` inside
+the opened YAML editor. If you have named your model **coolstore** as instructed,
+you're good to go. If not, modify the value on **line # 35** with the name you set.
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md:77]: will it always be line 35? can we point to something else unique on the row?

File: docs/openshift-ai/data-science-project/using-projects-the-rhoai.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
+# Using Your Data Science Project (DSP)
+
+You can access your current projects by navigating to the "Data Science Projects"
+menu item on the left-hand side, as highlighted in the figure below:
+
+![Data Science Projects](images/data-science-projects.png)
+
+If you have any existing projects, they will be displayed here. These projects
+corresponds to your [NERC-OCP (OpenShift) resource allocations](../../get-started/allocation/allocation-details.md#general-user-view-of-openshift-resource-allocation).
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/using-projects-the-rhoai.md:9]: corresponds -> correspond

File: docs/openshift-ai/data-science-project/using-projects-the-rhoai.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+under the status indicator and you can hover over that icon. Looking closely at
+the error message, it shows the details about the issue and you can resolve accordingly.
+
+When your workbench is ready, the state will change to *Running* and you can select
+"Open" to go to your environment:
+
+![Open JupyterLab Environment](images/open-jupyter-lab.png)
+
+!!! tip "How can I start or stop a Workbench?"
+    You can use this toggle to easily *start/stop* this environment later on.
```
Comment (joachimweyl) [inline docs/openshift-ai/data-science-project/using-projects-the-rhoai.md:115]: what toggle?

File: docs/openshift-ai/other-projects/configure-jupyter-notebook-use-gpus-aiml-modeling.md [hunk 15]
```diff
@@ -0,0 +1,377 @@
...
+
+- Select the correct data science project and create workbench, see
+[Populate the data science project](../data-science-project/using-projects-the-rhoai.md#populate-the-data-science-project)
+for more information.
+
+Please ensure that you start your Jupyter notebook server with options as depicted
+in the following configuration screen. This screen provides you with the opportunity
+to select a notebook image and configure its options, including the number of GPUs.
+
+![PyTorch Workbech Information](images/pytorch-workbench.png)
```
Comment (joachimweyl) [inline docs/openshift-ai/other-projects/configure-jupyter-notebook-use-gpus-aiml-modeling.md:15]: if you cut off the white space on the right the rest of the text will be bigger and easier to read.

File: docs/openshift-ai/other-projects/configure-jupyter-notebook-use-gpus-aiml-modeling.md [hunk 16]
```diff
@@ -0,0 +1,377 @@
...
+
+For our example project, let's name it "PyTorch Workbench". We'll select the
+**PyTorch** image, choose a **Deployment size** of **Small**, **Number of GPUs**
+as **1** and allocate a **Cluster storage** space of **1GB**.
+
+If this procedure is successful, you have started your Jupyter notebook server.
+When your workbench is ready, the state will change to *Running* and you can select
+"Open" to go to your environment:
+
+![Open JupyterLab Environment](images/open-jupyter-lab.png)
```
Comment (joachimweyl) [inline docs/openshift-ai/other-projects/configure-jupyter-notebook-use-gpus-aiml-modeling.md:25]: this image is too small to see, save it as a larger file and/or circle the area you are hovering over in red.

File: docs/openshift-ai/other-projects/how-access-s3-data-then-download-and-analyze-it.md [hunk 17]
```diff
@@ -0,0 +1,121 @@
...
+
+- Select the correct data science project and create workbench, see
+[Populate the data science project](../data-science-project/using-projects-the-rhoai.md#populate-the-data-science-project)
+for more information.
+
+Please ensure that you start your Jupyter notebook server with options as depicted
+in the following configuration screen. This screen provides you with the opportunity
+to select a notebook image and configure its options, including the number of GPUs.
+
+![Standard Data Science Workbech Information](images/standard-data-science-workbench.png)
```
Comment (joachimweyl) [inline docs/openshift-ai/other-projects/how-access-s3-data-then-download-and-analyze-it.md:15]: remove white from right side to make rest of image easier to read.

File: docs/openshift-ai/other-projects/how-access-s3-data-then-download-and-analyze-it.md [hunk 18]
```diff
@@ -0,0 +1,121 @@
...
+
+For our example project, let's name it "Standard Data Science Workbench". We'll
+select the **Standard Data Science** image, choose a **Deployment size** of **Small**,
+**Number of GPUs** as **0** and allocate a **Cluster storage** space of **1GB**.
+
+If this procedure is successful, you have started your Jupyter notebook server.
+When your workbench is ready, the state will change to *Running* and you can select
+"Open" to go to your environment:
+
+![Open JupyterLab Environment](images/open-jupyter-lab.png)
```
Comment (joachimweyl) [inline docs/openshift-ai/other-projects/how-access-s3-data-then-download-and-analyze-it.md:25]: made image larger and/or circle section in red.

**Review by joachimweyl — CHANGES_REQUESTED** [review]
requests inline.

File: docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md (hunk 12 above)
Comment (Milstein) [reply docs/openshift-ai/data-science-project/testing-model-in-the-rhoai.md:77]: Yes. Now I added link to the versioned github url with line # that will be fixed and not changed, even we change the code later time.

File: docs/openshift-ai/data-science-project/using-projects-the-rhoai.md (hunk 14 above)
Comment (Milstein) [reply docs/openshift-ai/data-science-project/using-projects-the-rhoai.md:115]: This is a **Toggle Switch** under the "Status" section.

**Review by joachimweyl — APPROVED** [review]

**Pull request merged**